ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -429,11 +431,15 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
+            worker_type=m5.4xlarge
```

Comment:
the hardcoded value below was m4.xlarge, not m5.  (not sure if there's a meaningful difference there, but i wouldn't want to change it unintentionally).